traitsui/qt4/constants.py

```diff
@@ -45,7 +45,12 @@
 
 # Screen size values:
 
-_geom = QtGui.QApplication.desktop().availableGeometry()
+# QDesktopWidget.availableGeometry(int screen) is deprecated and Qt docs
+# suggest using screens() instead, but screens in not available in qt4
+if QtCore.__version_info__ >= (5, 0):
+    _geom = QtGui.QApplication.screens()[0].availableGeometry()
+else:
+    _geom = QtGui.QApplication.desktop().availableGeometry()
```

Comment:
Can you open an issue to track and remove this compatibililty fix? I should have caught this earlier.